src/main/java/com/o19s/es/termstat/TermStatSupplier.java

```diff
@@ -68,7 +73,16 @@ public void bump (IndexSearcher searcher, LeafReaderContext context,
             assert termStates != null && termStates
                     .wasBuiltFor(ReaderUtil.getTopLevelContext(context));
 
-            TermState state = termStates.get(context);
```

Comment:
one can't get access to the state directly anymore, need to go through state supplier.